src/Microsoft.ML.DataView/VectorType.cs

```diff
@@ -21,6 +22,15 @@ public sealed class VectorDataViewType : StructuredDataViewType
         /// As with <see cref="Size"/>, a zero value indicates that the vector type is considered to have
         /// unknown length along that dimension.
         /// </summary>
+        /// <remarks>
+        /// In the case where this is a multi-dimensional type, that is, a situation where <see cref="Dimensions"/>
+        /// has length greater than one, since <see cref="VBuffer{T}"/> itself is a single dimensional sstructure,
```

Comment:
`sstructure`

Reply:
Thanksssssss my sss key isssss ssssstruck makesss writing the word "as" ssssssometimesss awkward.